README.md

```diff
@@ -6,6 +6,12 @@ Kontrolgruppen core bundle.
 
 Submodule of Kontrolgruppen. Run `yarn install` in your project root to build assets.
 
+## Environment
+Set the site name in the .env file:
+```ini
+site_name=your-site-name
```

Comment:
Should this not be
SITE_NAME
see https://github.com/aakb/kontrolgruppen/pull/56/files